src/content/docs/apm/apm-ui-pages/errors-inbox/errors-inbox.mdx

```diff
@@ -68,6 +68,8 @@ Clicking on a specific error group takes you to the error details page, where yo
 
```

Comment:
I would vote for updating the image (./images/context.png) above with one that includes the "Open in IDE" button